tests/goth_tests/test_multiactivity_agreement/test_multiactivity_agreement.py

```diff
@@ -39,7 +39,7 @@ async def assert_multiple_workers_run(agr_id, events):
         if m:
             worker_agr_id = m.group(1)
             assert worker_agr_id == agr_id, "Worker run for another agreement"
-            assert line.endswith(" exc_info=None)"), "Worker finished with error"
+            assert "exc_info=None" in line, "Worker finished with error"
```

Comment:
One of the integration tests return the following error:
> ERROR    [goth.assertions.monitor       ] [command output] Assertion 'assert_multiple_workers_run(4d61ce72ac360f030e5ef8ad4dbfaefd297f106bf357f1a08f2c9df816007ebb)' failed; cause: Worker finished with error
assert 'exc_info=None' in "WorkerFinished(job_id='1', agr_id='4d61ce72ac360f030e5ef8ad4dbfaefd297f106bf357f1a08f2c9df816007ebb')"

Maybe change this to:
```
assert "exc_info=" not in line, "Worker finished with error"
```
?